packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: unintended_html_in_doc_comment
```

Comment:
Could you add a comment here with a brief explanation, https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all-ignores